composer.json

```diff
@@ -184,6 +184,9 @@
             "drupal/message_digest": {
                 "Aggregated patch for issues #2285947 #2881263 #2933016 #2933896 #2933931 #2934111 @see https://www.drupal.org/project/issues/message_digest": "resources/patch/message_digest_aggregated.patch"
             },
+            "drupal/og": {
+                "OgRole::isAdmin() should return TRUE for administrators": "https://patch-diff.githubusercontent.com/raw/Gizra/og/pull/370.patch"
```

Comment:
As you comment in the case where this is used, we cannot use the `::isAdmin` method to ensure that a role is an administrator role. This patch is an improvement, but not needed.